media/js/app/projects/projectpanel.js

```diff
@@ -12,90 +13,123 @@ var ProjectPanelHandler = function (el, parent, panel, space_owner) {
     self.parentContainer = parent;
     self.space_owner = space_owner;
     self.tiny_mce_settings = tiny_mce_settings;
-    jQuery(self.el).find('.project-savebutton').attr("value", "Saved");
+    jQuery(self.el).find('.project-savebutton').attr('value', 'Saved');
 
     djangosherd.storage.json_update(panel.context);
+    MediaThread.loadTemplate('project_revisions');
 
     if (panel.context.can_edit) {
-        var select = jQuery(self.el).find("select[name='participants']")[0];
-        jQuery(select).addClass("selectfilter");
-        SelectFilter.init("id_participants_" + panel.context.project.id,
-            "participants", 0, "/media/admin/");
+        var select = jQuery(self.el).find('select[name="participants"]')[0];
+        jQuery(select).addClass('selectfilter');
+        SelectFilter.init('id_participants_' + panel.context.project.id,
+            'participants', 0, '/media/admin/');
 
         // HACK: move the save options around due to django form constraints
-        var assignment_elt = jQuery(self.el).find("label[for='id_publish_2']").parent();
+        var assignment_elt = jQuery(self.el).find('label[for="id_publish_2"]')
+                                            .parent();
         if (assignment_elt.length > 0) {
             jQuery(self.el).find('div.due-date').appendTo(assignment_elt);
         }
     }
 
     self.project_type = panel.context.project.project_type;
-    self.essaySpace = jQuery(self.el).find(".essay-space")[0];
+    self.essaySpace = jQuery(self.el).find('.essay-space')[0];
 
     // hook up behaviors
-    jQuery(window).on('tinymce_init_instance', function (event, instance, param2) {
-        self.onTinyMCEInitialize(instance);
-    });
+    jQuery(window).on('tinymce_init_instance',
+        function(event, instance, param2) {
+            self.onTinyMCEInitialize(instance);
+        });
 
-    jQuery(window).resize(function () {
+    jQuery(window).resize(function() {
         self.resize();
     });
 
-    self._bind(self.el, "td.panel-container", "panel_state_change", function () {
-        self.onClosePanel(jQuery(this).hasClass("subpanel"));
-    });
+    self._bind(self.el, 'td.panel-container', 'panel_state_change',
+        function() {
+            self.onClosePanel(jQuery(this).hasClass('subpanel'));
+        });
 
-    self._bind(self.el, "form[name='editproject']", 'keypress keydown keyup', function(e) {
-        if (e.keyCode == 13) {
-            e.preventDefault();
-        }
-    });
+    self._bind(self.el, 'form[name="editproject"]', 'keypress keydown keyup',
+        function(e) {
+            if (e.keyCode == 13) {
```

Comment:
good catch & fixed. I wonder why the jshint or jscs didn't catch that though...I'll have to look at configuration.